addons/payment/tests/test_flows.py

```diff
@@ -347,3 +326,44 @@ def test_transaction_wrong_token(self):
         self.assertIn(
             "odoo.exceptions.ValidationError: The access token is invalid.",
             response.text)
+
+    @mute_logger('odoo.addons.payment.models.payment_transaction')
+    def _test_max_one_payment_request_from_portal(self, flow):
+
+        payment_option_id = self.create_token().id if flow == 'token' else self.acquirer.id
+        data = {
+            'amount': self.amount,
+            'currency_id': self.currency.id,
+            'partner_id': self.partner.id,
+            'access_token': self._generate_test_access_token(
+                self.partner.id, self.amount, self.currency.id
+            ),
+            'payment_option_id': payment_option_id,
+            'reference_prefix': 'test',
+            'tokenization_requested': True,
+            'landing_route': 'Test',
+            'is_validation': False,
+            'invoice_id': self.invoice.id,
+            'flow': flow,
+        }
+
+        with patch(
+            'odoo.addons.payment.models.payment_transaction.PaymentTransaction.'
+            '_send_payment_request'
+        ) as patched:
+
+            response = self.portal_transaction(**data)
+            self.assertEqual(response.status_code, 200)
```

Comment:
cfr https://github.com/odoo/odoo/pull/80590#discussion_r774452063
:slightly_smiling_face:

Reply:
Or I could use `get_processing_values` instead of `portal_transaction`, might feel useless since I don't care about the response content, but at least, the status code check would be made without adding noise to the tests file

Reply:
No, testing for the response code is just fine, and calling `portal_transaction` clearly shows what you're trying to achieve here.

Instead, you could simply add a comment like:
```py
self.assertEqual(response.status_code, 200)  # First, check the request went through.
```
Or, even better, move that check inside `portal_transaction` and similar helpers so that we can always rely on the response.